src/components/map/layers/NodeLayer.tsx

```diff
@@ -7,7 +7,6 @@ import { INode } from '~/models';
 import NodeType from '~/enums/nodeType';
 import { PopupStore } from '~/stores/popupStore';
 import { ToolbarStore } from '~/stores/toolbarStore';
-import { SidebarStore } from '~/stores/sidebarStore';
```

Comment:
Sidebar was only used by linkview, reference to sidebar store was removed, so removing a lot of old references to sidebar store in the project